### PR TITLE
Add vaccine selector for average people

### PR DESCRIPTION
**Pull request by beshaya**
- Add vaccine selector for other person
- Add vaccine option for average risk profile

### CODE REVIEW

**Comment by blanchardjeremy** [comment]
I'm so excited this is so close to being live. 

One usability issue: It's a bit redundant of a user experience on mobile. It says the same text in three different places right on top of each other. 


0B92CAF7-02A8-4307-BB8E-AD946549A1B5

**Comment by beshaya** [comment]
> One usability issue: It's a bit redundant of a user experience on mobile. It says the same text in three different places right on top of each other.

Removed the description showing up in the label. I think you're testing on a laptop with narrow width, which is how you get the tooltip; on mobile there's no mouse so I think that tooltip never shows up.

**Comment by blanchardjeremy** [comment]
> on mobile there's no mouse so I think that tooltip never shows up.

Actually, the tap action does activate the tooltip on mobile. That was an iOS screenshot. You might wanna pull it up and see how it is on your device.

**Comment by beshaya** [comment]
> > on mobile there's no mouse so I think that tooltip never shows up.
> 
> Actually, the tap action does activate the tooltip on mobile. That was an iOS screenshot. You might wanna pull it up and see how it is on your device.

TIL. I think I fixed it.

**Comment by blanchardjeremy** [comment]
It's still doing the.hover thing. For the case of unsure, the hover text blocks the main text. 

D3B68178-C899-44A3-A139-34236A4C8D5C
00670383-AAF0-405C-9BB8-7AAC8A224332

**Comment by blanchardjeremy** [comment]
Although to be fair, the mask selector does the same thing, so I think we could consider that "issue" non-blocking for this PR.

File: src/posts/paper/14-research-sources.ts [hunk 1]
```diff
@@ -423,6 +423,38 @@ For vaccines require 2 doses, we see reductions in symptomatic COVID after a sin
...
+\`\`\`
+Average prevalence = sum(prevalence in group * proportion of population in group)
+                   = sum(vaccine_mult * unvaccinated_prev * proportion of population)
+Unvaccinated risk / Average risk = sum(vaccine_mult * proportion of pop)
+                                 = population / sum(vaccine_mult * proportion of pop)
+Where the sum is taken over all vaccine types and status (including no vaccine)
+\`\`\`
+
+To get data for vaccine prevalence, we use data from JHU for Countires and US States,
+combined with data from Covid Act Now for vaccines in a county. JHU breakds down
```
Comment (jy2wong) [inline src/posts/paper/14-research-sources.ts:446]: breaks

File: src/posts/paper/14-research-sources.ts [hunk 2]
```diff
@@ -423,6 +423,38 @@ For vaccines require 2 doses, we see reductions in symptomatic COVID after a sin
...
+Where the sum is taken over all vaccine types and status (including no vaccine)
+\`\`\`
+
+To get data for vaccine prevalence, we use data from JHU for Countires and US States,
+combined with data from Covid Act Now for vaccines in a county. JHU breakds down 
+vaccinations by manufacturer for US States, and we assume that US Counties have the 
+same ratio of vaccine types.
+
+We do not have a data source for vaccine type breakdown in countries outside the US; 
+we calculate as if all vaccines were AstraZenica. This results in under-estimating the
```
Comment (jy2wong) [inline src/posts/paper/14-research-sources.ts:451]: AstraZeneca

File: src/components/calculator/controls/SegmentedControl.tsx [hunk 3]
```diff
@@ -54,13 +57,15 @@ export const SegmentedControl: React.FunctionComponent<{
   const hoverDescIfPresentOtherwiseActiveDesc =
     hoverDesc !== ''
       ? hoverDesc
-      : (activeValue !== '' && props.source[activeValue].value) || ''
+      : props.useActiveDesc && activeValue !== ''
```
Comment (jy2wong) [inline src/components/calculator/controls/SegmentedControl.tsx:60]: I find nested ternaries a little hard to read, but I'm not fussed enough to insist it changes.

**Pull request merged**